src/hooks/useQueryList.jsx

```diff
@@ -34,37 +28,43 @@ export const useQueryList = function (
   }, [limit]);
 
   useEffect(() => {
-    const client = admin ? spvWalletAdminClient : spvWalletClient;
-    if (!client) return;
-    client[`${modelFunction}Count`](conditions || {}, {}).then(count => {
-      setItemsCount(count);
-      setError('');
-    }).catch(e => {
-      logger.error(e)
-      setItemsCount(limit);
-      setError(e.message);
-    });
+    if (!spvWalletClient) {
+      return;
+    }
+    spvWalletClient[`${modelFunction}Count`](conditions || {}, {})
+      .then((count) => {
+        setItemsCount(count);
+        setError('');
+      })
+      .catch((e) => {
+        logger.error(e);
+        setItemsCount(limit);
+        setError(e.message);
+      });
   }, [refreshData, conditions]);
 
   useEffect(() => {
-    setLoading(true)
+    setLoading(true);
     const queryParams = {
       page: page + 1,
       page_size: limit,
       order_by_field: 'created_at',
       sort_direction: 'desc',
     };
-    const client = admin ? spvWalletAdminClient : spvWalletClient;
-    if (!client) return;
-    client[`${modelFunction}`](conditions || {}, {}, queryParams).then(items => {
-      setItems([...items]);
-      setError('');
-      setLoading(false);
-    }).catch(e => {
-      logger.error(e)
-      setError(e.message);
-      setLoading(false);
-    });
+    if (!spvWalletClient) {
+      return spvWalletClient;
+    }
```

Comment:
I don't get this 😄  We should just `return` nothing here I think

Reply:
I don't get it too 🙈 
I've changed to "return nothing"